front/lib/utils.ts

```diff
@@ -291,5 +303,5 @@ function alphanumFromByte(byte: number) {
   if (index < 62) {
     return CHAR_0 + index - 52;
   }
-  return index === 62 ? 45 : 95;
+  return 0;
```

Comment:
Null characters are preserved in the string representation even if they don't count for the length no?

Reply:
They actually count for the length as well

```
> new Buffer([62,0,0,64]).toString()
'>\x00\x00@'
> new Buffer([62,0,0,64]).toString().length
4
```

Reply:
:see_no_evil: will fix

Reply:
Could either do a theoretically better code with a carry param & a reduce, or just filter the null string
Went for the second that is IMHO easier to follow